paasta_tools/paasta_metastatus.py

```diff
@@ -349,8 +350,12 @@ def main():
         chronos_results = [('chronos is not configured to run here', True)]
 
     if marathon_config:
-        marathon_client = get_marathon_client(marathon_config)
-        marathon_results = get_marathon_status(marathon_client)
+        try:
+            marathon_client = get_marathon_client(marathon_config)
+            marathon_results = get_marathon_status(marathon_client)
```

Comment:
only wrap ``marathon_results = get_marathon_status(marathon_client)`` in the try/except here - creating the client doesn't throw an exception.